src/collective/volto/formsupport/browser/send_mail_template.pt

```diff
@@ -11,7 +16,12 @@
       </li>
     </tal:field>
   </ul>
-  <p i18n:translate="send_mail_text">
-    A new form has been submitted from <a href="${url}"><strong i18n:name="url">${url}</strong></a>
-  </p>
+  <tal:element tal:condition="python:not block.get('mail_footer', '')">
```

Comment:
why does this is not inside a <footer> tag if is the default when it's not set?

Reply:
what do u mean, example?